.github/workflows/test.yml

```
@@ -40,7 +40,7 @@ jobs:
     strategy:
       matrix:
         os: [ ubuntu-latest, windows-latest, macos-latest ]
-        java: [ 8, 11, 16 ]
+        java: [ 8, 8.0.192, 11, 11.0.3, 16 ]
```

No need to add latest head versions of JDK here. Testing against the latest stable versions is preferred:
```suggestion
        java: [ 8, 11, 16 ]
```